FastSimulation/ForwardDetectors/plugins/FastSimDataFilter.cc

```diff
@@ -3,38 +3,35 @@
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/Framework/interface/ESHandle.h"
 #include "FWCore/PluginManager/interface/ModuleDef.h"
 #include "FWCore/ServiceRegistry/interface/Service.h"
 #include "DataFormats/DetId/interface/DetId.h"
 #include "DataFormats/CaloTowers/interface/CaloTowerDetId.h"
-#include "DataFormats/CaloTowers/interface/CaloTowerCollection.h"
 
 using namespace std;
 namespace cms {
 
-  FastSimDataFilter::FastSimDataFilter(const edm::ParameterSet& conf) {
-    towercut = conf.getUntrackedParameter<double>("towercut", 1.4);
-  }
-
-  FastSimDataFilter::~FastSimDataFilter() {}
+  FastSimDataFilter::FastSimDataFilter(const edm::ParameterSet& conf)
+      : tokTowers_(consumes<CaloTowerCollection>(edm::InputTag("towerMaker"))),
+        towercut(conf.getUntrackedParameter<double>("towercut", 1.4)) {}
 
   void FastSimDataFilter::beginJob() {
     ntotal = 0;
     npassed = 0;
   }
   void FastSimDataFilter::endJob() {
-    cout << " FastSimDataFilter: accepted " << npassed << "  out of total " << ntotal << endl;
+    edm::LogVerbatim("FastSim") << " FastSimDataFilter: accepted " << npassed << "  out of total " << ntotal;
```

Comment:
Writing the name of the class, instead of a generic "FastSim" can help in case of using these messages for debug:
```suggestion
    edm::LogVerbatim("FastSimDataFilter") << " FastSimDataFilter: accepted " << npassed << "  out of total " << ntotal;
```